apollo-router-core/well_known_introspection_queries/npx_diagnose_endpoint1.graphql

```diff
@@ -0,0 +1 @@
+query Ping { __typename }
```

Comment:
:yarn: It would be best if we can avoid the missing EOF newlines when using Git

Reply:
It would indeed ^^, however we're not normalizing queries yet, so it has to exactly match what is being sent by the clients

Reply:
oh that rings a bell :sweat_smile: sorry!